homework5-3.py

```diff
@@ -0,0 +1,24 @@
+# n = 2
+# Russia Moscow Petersburg Novgorod Kaluga
+# Ukraine Kiev Donetsk Odessa
+# m = 3
+# city1 = Odessa
+# city2 = Moscow
+# city3 = Novgorod
+
+n = int(input('Enter N: '))
+dct = {}
+for i in range(n):
+    text = str(input('Enter country and cities: '))
```

Comment:
it can be used like 
country, *cities = input('counties and cities').split() and cities would be put to cities tuple